src/schema/root.js

```diff
@@ -69,7 +69,7 @@ type Query {
 
 
   help(q: String!, language: LanguageCode): HelpResponse
-  branches(agencyid: String, branchId: String, language: LanguageCode, q: String, offset: Int, limit: PaginationLimit, status: LibraryStatus): BranchResult!
+  branches(agencyid: String, branchId: String, language: LanguageCode, q: String, offset: Int, limit: PaginationLimit, status: LibraryStatus, excludebranches: Boolean): BranchResult!
```

Comment:
Det vil være synd at kalde den exlude branches - for hvad betyder det?
infomediaAccess: true|false
digitalCopyAccess: true|false

Reply:
og pickupAllowed: true|false